mongo_orchestration/apps/__init__.py

```diff
@@ -45,7 +47,9 @@ def setup_versioned_routes(routes, version=None):
 def send_result(code, result=None):
     response.content_type = None
     if result is not None and 200 <= code < 300:
-        result = json.dumps(result)
+        # Use json.dumps instead of json_util.dumps to only encode
+        # non JSON serializable BSON types with json_util.
+        result = json.dumps(result, default=json_util.default)
```

Comment:
The reason I was hesitant to use `json_util` is if the default output changes to canonical extended json. In that case, MO's responses would change from `..."port": 27017...` to `..."port": {"$numberInt": "27017"}...`. Doing this avoids changing MO's responses if/when PyMongo changes.